Update the Governance section of the website for the ROS PMC. (#4589)
ros2/ros2_documentation: source/The-ROS2-Project/Governance.rst

==================

.. contents:: Table of Contents
   :depth: 2
   :local:

The Open Source Robotics Alliance (OSRA)
----------------------------------------

Since 2024, the ROS 2 project has been governed by the `Open Source Robotics Alliance (OSRA) <https://osralliance.org/>`__.

> The information below is meant to give a quick overview of the project governance, but for full information please see https://osralliance.org/how-it-works/.

Reviewer: ```suggestion
The information below is meant to give a quick overview of the project governance, but for full information please see `the OSRA's website <https://osralliance.org/how-it-works/>`.
```

The Technical Governance Committee (TGC)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

The Technical Governance Committee is responsible for the oversight of all projects within the OSRA.
The TGC is made up of a combination of paid members, project management leaders, OSRF leaders, and members based on merits.

> For more details about the TGC, please see https://osralliance.org/staging/wp-content/uploads/2024/03/OSRA-Program-Charter.pdf.

Reviewer: ```suggestion
For more details about the TGC, please see `the Charter for the OSRA <https://osralliance.org/staging/wp-content/uploads/2024/03/OSRA-Program-Charter.pdf>`.
```

> One of the projects that the TGC oversees is the ROS Project Management Committee, which governs ROS 2.

Reviewer: ```suggestion
One of the projects that the TGC oversees is ROS 2, which is managed by the ROS Project Management Committee.
```

The ROS Project Management Committee (ROS PMC)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

The ROS Project Management Committee is responsible for the day-to-day operations of the ROS 2 project.

> The ROS PMC consists of a project leader, members (which have full voting rights) and committers (which can attend meetings but do not have voting rights).

Reviewer: ```suggestion
The ROS PMC consists of the Project Leader, the ROS PMC Members (who have full voting rights), a Supporting Individual Representative, and the Chair of the TGC. The project also has Committers, who help manage one or more repositories but are not a part of the PMC.
```

> The leader, all members, and all committers are chosen on a meritocratic basis.

Reviewer: ```suggestion
The Project Leader, all PMC Members, and all Committers are chosen on a meritocratic basis.
```

> THe more effort a person puts into ROS 2, the more influence they may have on the ROS PMC.

Reviewer: ```suggestion
```

The day-to-day operations of the ROS PMC include managing the members and committers, managing the repositories that make up ROS 2, reviewing and merging code from the ROS community, maintaining the repositories, and making technical decisions that decide the direction of the project.

> For more details about the ROS PMC, please see https://osralliance.org/staging/wp-content/uploads/2024/03/ros_project_charter.pdf

Reviewer: ```suggestion
For more details about the ROS PMC, please see the `Charter for the ROS Project <https://osralliance.org/staging/wp-content/uploads/2024/03/ros_project_charter.pdf>`
```
